src/Chorus/UI/Settings/SendReceiveSettings.resx

```diff
@@ -1,141 +1,142 @@
+          </xsd:element>
+          <xsd:element name="resheader">
+            <xsd:complexType>
+              <xsd:sequence>
+                <xsd:element name="value" type="xsd:string" minOccurs="0" msdata:Ordinal="1" />
+              </xsd:sequence>
+              <xsd:attribute name="name" type="xsd:string" use="required" />
+            </xsd:complexType>
+          </xsd:element>
+        </xsd:choice>
+      </xsd:complexType>
+    </xsd:element>
   </xsd:schema>
   <resheader name="resmimetype">
-	<value>text/microsoft-resx</value>
+    <value>text/microsoft-resx</value>
   </resheader>
   <resheader name="version">
-	<value>2.0</value>
+    <value>2.0</value>
   </resheader>
   <resheader name="reader">
-	<value>System.Resources.ResXResourceReader, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
+    <value>System.Resources.ResXResourceReader, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
   </resheader>
   <resheader name="writer">
-	<value>System.Resources.ResXResourceWriter, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
+    <value>System.Resources.ResXResourceWriter, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
   </resheader>
-  <metadata name="l10NSharpExtender1.TrayLocation" type="System.Drawing.Point, System.Drawing, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a">
-	<value>143, 17</value>
-  </metadata>
-  <data name="chorusHubSetup.DocumentText" xml:space="preserve">
-	<value>&lt;!DOCTYPE html&gt;&lt;html&gt;&lt;body style='font-family:&quot;Microsoft Sans Serif&quot;,&quot;sans-serif&quot;;font-size:9pt'&gt;
+  <assembly alias="System.Drawing" name="System.Drawing, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a" />
+  <data name="l10NSharpExtender1.TrayLocation" type="System.Drawing.Point, System.Drawing">
+    <value>143, 17</value>
+  </data>
+  <data name="_helpProvider.TrayLocation" type="System.Drawing.Point, System.Drawing">
+    <value>17, 17</value>
+  </data>
+  <data name="ChorusHubSetupInstructionsHTML" xml:space="preserve">
+    <value>&lt;!DOCTYPE html&gt;&lt;html&gt;&lt;body style='font-family:"Microsoft Sans Serif","sans-serif";font-size:9pt'&gt;
 &lt;p style='margin:0in;'&gt;Chorus Hub is a simple way to Send/Receive on a local network, like in an office or home. These instructions are basic, but they should be enough to get you started.&lt;/p&gt;
 &lt;ol&gt;
-&lt;li style='margin-left:-10pt'&gt;Designate one computer on your network as &quot;the server&quot;, and use the following instructions to make sure this product is installed and running on it.&lt;/li&gt;
+&lt;li style='margin-left:-10pt'&gt;Designate one computer on your network as "the server", and use the following instructions to make sure this product is installed and running on it.&lt;/li&gt;
```

Comment:
Well, VisualStudio did the unescaping without my input, and it appears that the content displays correctly. Quotes in html content don't need to be escaped.